### PR TITLE
EZP-32320: Fixed image placeholders generation on DFS

### PR DESCRIPTION
**Pull request by adamwojs**
| Question           | Answer
| ------------------ | ------------------
| **JIRA issue**     | [EZP-32320](https://jira.ez.no/browse/EZP-32320)
| **Bug/Improvement**| yes
| **New feature**    | no
| **Target version** | `7.5`
| **BC breaks**      | no
| **Tests pass**     | yes
| **Doc needed**     | yes

Added `verify_binary_data_availability` configuration option to enable additional binary data availability check in `\eZ\Bundle\EzPublishCoreBundle\Imagine\PlaceholderAliasGenerator` in order to detect missing images. By default binary data availability check is disabled to avoid additional IO operations.

**TODO**:
- [x] Implement feature / fix a bug.
- [x] Implement tests.
- [x] Fix new code according to Coding Standards (`$ composer fix-cs`).
- [x] Ask for Code Review.

### CODE REVIEW

File: eZ/Bundle/EzPublishCoreBundle/DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -618,6 +618,9 @@ private function addImagePlaceholderSection(ArrayNodeDefinition $rootNode)
                             ->variableNode('options')
                                 ->defaultValue([])
                             ->end()
+                            ->booleanNode('verify_binary_data_availability')
+                                ->defaultFalse()
+                            ->end()
```
Comment (Steveb-p) [inline eZ/Bundle/EzPublishCoreBundle/DependencyInjection/Configuration.php]: A description of the side-effects of enabling this option would be great here.

https://symfony.com/doc/current/components/config/definition.html#documenting-the-option

```suggestion
                            ->booleanNode('verify_binary_data_availability')
                                ->defaultFalse()
                                ->info('???')
                            ->end()
```

Comment (adamwojs) [reply eZ/Bundle/EzPublishCoreBundle/DependencyInjection/Configuration.php]: Fixed in eddb699.

**Review by Steveb-p — APPROVED** [review]
LGTM

**Review by ViniTou — APPROVED** [review]

**Review by alongosz — APPROVED** [review]

**Review by tomaszszopinski — APPROVED** [review]
QA approved on ezplat-ee 2.5 & Ibexa dxp 3.2.4 & 3.3.1-dev with diff & patch.

**Comment by lserwatka** [comment]
Could you merge it up?

**Pull request merged**